gradle/dependency-locks/embulkPluginRuntime.lockfile

```diff
@@ -1,7 +1,12 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
+com.fasterxml.jackson.core:jackson-core:2.6.7
```

Comment:
Does it have only `jackson-core`? It should have other dependencies the same with `embulk-util-config`.

Reply:
Thanks. Fixed.